cmd/helmvm/install.go

```diff
@@ -412,50 +443,33 @@ var installCommand = &cli.Command{
 			metrics.ReportApplyFinished(c, err)
 			return err
 		}
-		if !c.Bool("addons-only") {
```

Comment:
Let's remove the `addons-only` flag altogether ? It seems like we don't need it anymore. What do you think ? Remember to check if it is not referenced in the README.